build/template-run-integration-tests.yaml

```diff
@@ -19,11 +19,25 @@ steps:
     pathtoCustomTestAdapters: 'C:\temp' # Workaround for test failure, as NUnit Test Adapter that gets detected seems to screw something up
 
 - task: VSTest@2
-  displayName: 'Run integration tests (.NET Core)'
+  displayName: 'Run integration tests (.NET Core 3.1)'
```

Comment:
Do we need .NET Core 3.1 integration tests? Can't we rely just on unit tests being run there?

Reply:
Yea, I think Net Core can be removed. NetFx tests would cover Newtonsoft flow, Net6 tests would cover STJ flow + existing non-NetFx NetCore tests.